internal/worker/server_test.go

```diff
@@ -142,6 +142,43 @@ func TestUpdate(t *testing.T) {
 	test.TestRoute(t, handler, false, "PATCH", fmt.Sprintf("/api/worker/v1/jobs/%s", token), `{}`, http.StatusNotFound, `*`)
 }
 
+func TestArgs(t *testing.T) {
+	distroStruct := fedoratest.New()
+	arch, err := distroStruct.GetArch("x86_64")
+	if err != nil {
+		t.Fatalf("error getting arch from distro")
+	}
+	imageType, err := arch.GetImageType("qcow2")
+	if err != nil {
+		t.Fatalf("error getting image type from arch")
+	}
+	manifest, err := imageType.Manifest(nil, distro.ImageOptions{Size: imageType.Size(0)}, nil, nil, nil, 0)
+	if err != nil {
+		t.Fatalf("error creating osbuild manifest")
+	}
+	server := worker.NewServer(nil, testjobqueue.New(), "")
+	server.Handler()
+
+	job := worker.OSBuildJob{
+		Manifest:  manifest,
+		ImageName: "test-image",
+	}
+	jobId, err := server.EnqueueOSBuild(arch.Name(), &job)
+	require.NoError(t, err)
+
+	_, _, _, args, _, err := server.RequestJob(context.Background(), arch.Name(), []string{"osbuild"})
+	require.NoError(t, err)
+	require.NotNil(t, args)
+
+	var jobArgs worker.OSBuildJob
+	rawArgs, err := server.JobArgs(jobId, &jobArgs)
+	if err != nil {
```

Comment:
I suggest using `require.NoError()` (there are in total 4 cases of `if err != nil`).